### PR TITLE
Fix doc build warning

### PR DESCRIPTION
**Pull request by leckie-chn**
WARNINGs left:

```
WARNING: autodoc: failed to import class 'smac_tuner.SMACTuner' from module 'nni'; the following exception was raised:
No module named 'smac'
WARNING: autodoc: failed to import class 'ppo_tuner.PPOTuner' from module 'nni'; the following exception was raised:
No module named 'gym'
WARNING: autodoc: failed to import class 'bohb_advisor.BOHB' from module 'nni.bohb_advisor'; the following exception was raised:
No module named 'ConfigSpace'
looking for now-outdated files... none found
pickling environment... done
checking consistency... /home/yann/nni/docs/en_US/Release_v1.0.md: WARNING: document isn't included in any toctree
```

For docstring formatting, please refer to `nni.msg_dispatcher_base.MsgDispatcherBase.handle_request_trial_jobs` where I wrote an example about how to format the docstring better. @xuehui1991 

image

### CODE REVIEW

File: docs/en_US/TrialExample/RocksdbExamples.md [hunk 1]
```diff
@@ -8,9 +8,9 @@ The performance of RocksDB is highly contingent on its tuning. However, because
...
 
-We also provide a simple script [`db_bench_installation.sh`](../../../examples/trials/systems/rocksdb-fillrandom/db_bench_installation.sh) helping to compile and install `db_bench` as well as its dependencies on Ubuntu. Installing RocksDB on other systems can follow the same procedure.
+We also provide a simple script [`db_bench_installation.sh`](https://github.com/microsoft/nni/tree/master/examples/trials/systems/rocksdb-fillrandom/db_bench_installation.sh) helping to compile and install `db_bench` as well as its dependencies on Ubuntu. Installing RocksDB on other systems can follow the same procedure.
 
-*code directory: [`example/trials/systems/rocksdb-fillrandom`](../../../examples/trials/systems/rocksdb-fillrandom)*
+*code directory: [`example/trials/systems/rocksdb-fillrandom`](https://github.com/microsoft/nni/tree/master/../../../examples/trials/systems/rocksdb-fillrandom)*
```
Comment (QuanluZhang) [inline docs/en_US/TrialExample/RocksdbExamples.md]: "../../../"?

**Comment by xuehui1991** [comment]
Comments in teams.

**Comment by leckie-chn** [comment]
whether to remove these following document
- [ ] /home/yann/nni/docs/en_US/AdvancedFeature/AdvancedNas.md
- [ ] /home/yann/nni/docs/en_US/AdvancedFeature/GeneralNasInterfaces.md
- [ ] /home/yann/nni/docs/en_US/CommunitySharings/TuningSystems.md
- [ ] /home/yann/nni/docs/en_US/Release_v1.0.md
- [ ] /home/yann/nni/docs/en_US/SupportedFramework_Library.md
- [ ] /home/yann/nni/docs/en_US/TrainingService/SupportTrainingService.md
- [ ] /home/yann/nni/docs/en_US/TrialExample/RocksdbExamples.md
- [ ] /home/yann/nni/docs/en_US/Tuner/PPOTuner.md

**Comment by xuehui1991** [comment]
I think we should not put "https://github.com/microsoft/nni/tree/master"

File: docs/en_US/TrialExample/RocksdbExamples.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ In this example, the search space is specified by a `search_space.json` file as
 }
 ```
 
-*code directory: [`example/trials/systems/rocksdb-fillrandom/search_space.json`](../../../examples/trials/systems/rocksdb-fillrandom/search_space.json)*
+*code directory: [`example/trials/systems/rocksdb-fillrandom/search_space.json`](https://github.com/microsoft/nni/tree/master/examples/trials/systems/rocksdb-fillrandom/search_space.json)*
```
Comment (xuehui1991) [inline docs/en_US/TrialExample/RocksdbExamples.md]: When we develop in dev-branch, the link is not in the master branch. So this link `https://github.com/microsoft/nni/tree/master` might be not valid during developing.

Comment (leckie-chn) [reply docs/en_US/TrialExample/RocksdbExamples.md]: It is valid in `master` now. BTW, we do not know which dev branch to link to.

File: docs/en_US/TrialExample/RocksdbExamples.md (hunk 1 above)
Comment (leckie-chn) [reply docs/en_US/TrialExample/RocksdbExamples.md]: fixed

File: docs/en_US/Release.md [hunk 3]
```diff
@@ -153,7 +153,7 @@
   * Run trial jobs on the GPU running non-NNI jobs
 * Kubeflow v1beta2 operator
   * Support Kubeflow TFJob/PyTorchJob v1beta2
-* [General NAS programming interface](AdvancedFeature/GeneralNasInterfaces.md)
+* *General NAS programming interface (deprecated)*
```
Comment (QuanluZhang) [inline docs/en_US/Release.md]: no need to change this part

File: docs/en_US/Release.md [hunk 4]
```diff
@@ -266,7 +266,7 @@
 * Support [Metis tuner](Tuner/MetisTuner.md) as a new NNI tuner. Metis algorithm has been proofed to be well performed for **online** hyper-parameter tuning.
 * Support [ENAS customized tuner](https://github.com/countif/enas_nni), a tuner contributed by github community user, is an algorithm for neural network search, it could learn neural network architecture via reinforcement learning and serve a better performance than NAS.
 * Support [Curve fitting assessor](Assessor/CurvefittingAssessor.md) for early stop policy using learning curve extrapolation.
-* Advanced Support of [Weight Sharing](AdvancedFeature/AdvancedNas.md): Enable weight sharing for NAS tuners, currently through NFS.
+* Advanced Support of *Weight Sharing*: Enable weight sharing for NAS tuners, currently through NFS.
```
Comment (QuanluZhang) [inline docs/en_US/Release.md]: same here

File: docs/en_US/TrialExample/RocksdbExamples.md [hunk 5]
```diff
@@ -87,7 +87,7 @@ We ran these two examples on the same machine with following details:
 
 The detailed experiment results are shown in the below figure. Horizontal axis is sequential order of trials. Vertical axis is the metric, write OPS in this example. Blue dots represent trials for tuning RocksDB with SMAC tuner, and orange dots stand for trials for tuning RocksDB with TPE tuner. 
 
-![image](../../../examples/trials/systems/rocksdb-fillrandom/plot.png)
+![image](https://github.com/microsoft/nni/tree/master/examples/trials/systems/rocksdb-fillrandom/plot.png)
```
Comment (QuanluZhang) [inline docs/en_US/TrialExample/RocksdbExamples.md]: which warning is it for this fix?

File: docs/en_US/Compressor/trial.py [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+                    'frequency': 1,'op_name': 'conv0' },
+                   {'initial_sparsity': 0, 'final_sparsity': conv1_sparsity,
+                    'start_epoch': 0, 'end_epoch': 3,
+                    'frequency': 1,'op_name': 'conv1' },]
+PRUNERS = {'level':LevelPruner(model, config_list_level), 'agp':AGP_Pruner(model, config_list_agp)}
+pruner = PRUNERS(params['prune_method']['_name'])
+pruner.compress()
+# fine tuning
+acc = evaluate(model) # evaluation
+nni.report_final_results(acc)
```
Comment (QuanluZhang) [inline docs/en_US/Compressor/trial.py]: where is this file used?

Comment (leckie-chn) [reply docs/en_US/Compressor/trial.py]: seems a mistake when merging with master... will remove it.

File: docs/en_US/TrialExample/RocksdbExamples.md (hunk 5 above)
Comment (leckie-chn) [reply docs/en_US/TrialExample/RocksdbExamples.md]: deadlink warning.

File: docs/en_US/Release.md (hunk 3 above)
Comment (leckie-chn) [reply docs/en_US/Release.md]: work around in 31788b692de591a6557579ce203b8838c32b424c
